app/styles/_base.scss

```diff
@@ -1,14 +1,8 @@
 @import '../bower_components/normalize-scss/normalize';
-@import url(http://fonts.googleapis.com/css?family=Fira+Sans:400,700,400italic,700italic);
+@import url(https://fonts.googleapis.com/css?family=Fira+Sans:400,500,700,400italic,700italic);
```

Comment:
Yeah, I didn't at first because I was afraid sass would think it was a comment. Seems to be fine. Pushing now...